patterns/footer-portfolio.php

```diff
@@ -19,18 +19,18 @@
 <!-- wp:columns {"style":{"spacing":{"padding":{"top":"2vh"}}}} -->
 <div class="wp-block-columns" style="padding-top:2vh"><!-- wp:column {"width":"57%"} -->
 <div class="wp-block-column" style="flex-basis:57%"><!-- wp:paragraph {"fontSize":"x-large","fontFamily":"cardo"} -->
-<p class="has-cardo-font-family has-x-large-font-size">Resta al passo, contattaci.</p>
+<p class="has-cardo-font-family has-x-large-font-size"><?php echo esc_html_x( 'Resta al passo, contattaci.', 'Portfolio Title', 'twentytwentyfour' ); ?></p>
 <!-- /wp:paragraph --></div>
 <!-- /wp:column -->
 
 <!-- wp:column {"width":"30%"} -->
 <div class="wp-block-column" style="flex-basis:30%"><!-- wp:group {"style":{"spacing":{"blockGap":"2px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
 <div class="wp-block-group"><!-- wp:paragraph -->
-<p>Contattami</p>
+<p><?php echo esc_html__( 'Contattami', 'twentytwentyfour' ); ?></p>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p><a href="#">example@wordpress.org</a></p>
+<p><a href="<?php echo esc_url( 'mailto:example@wordpress.org', 'twentytwentyfour' ); ?>"><?php echo esc_html( 'example@wordpress.org', 'twentytwentyfour' ); ?></a></p>
```

Comment:
Please remove this, so that people don't start emailing this address, by mistake or otherwise.

Reply:
I have removed maito tag and added the # now.